pkg/sdk/warehouse_test.go

```diff
@@ -95,20 +84,19 @@ func TestWarehouseAlter(t *testing.T) {
 	t.Run("with unset", func(t *testing.T) {
 		opts := &WarehouseAlterOptions{
 			name: NewAccountObjectIdentifier("mywarehouse"),
-			Unset: &[]WarehouseUnsetField{
-				WarehouseSizeField,
-				MaxClusterCountField,
-				AutoResumeField,
-				// Tag: []ObjectIdentifier{
-				// 	NewSchemaObjectIdentifier("db1", "schema1", "tag1"),
-				// 	NewSchemaObjectIdentifier("db2", "schema2", "tag2"),
-				// },
+			Unset: &WarehouseUnset{
+				WarehouseSize:   Bool(true),
+				MaxClusterCount: Bool(true),
+				AutoResume:      Bool(true),
+				Tag: []ObjectIdentifier{
+					NewSchemaObjectIdentifier("db", "schema", "tag1"),
+				},
 			},
 		}
 		clauses, err := builder.parseStruct(opts)
 		require.NoError(t, err)
 		assert.Equal(t,
-			`ALTER WAREHOUSE "mywarehouse" UNSET WAREHOUSE_SIZE,MAX_CLUSTER_COUNT,AUTO_RESUME`,
+			`ALTER WAREHOUSE "mywarehouse" UNSET WAREHOUSE_SIZE,MAX_CLUSTER_COUNT,AUTO_RESUME,TAG "db"."schema"."tag1"`,
```

Comment:
This is invalid SQL, cf previous comment about SET + TAG.

Reply:
![image](https://github.com/Snowflake-Labs/terraform-provider-snowflake/assets/128428373/a57f3e33-4e2a-45e2-9d2f-d7acc6453546)